README.md

```diff
@@ -4,31 +4,23 @@
 
 Browse the developer documentation to start integrating blockchain technology into your project.
 
-## Symbol is Your Blockchain Platform
+## Symbol is built for developers
 
-Symbol is a developer-friendly blockchain platform for businesses.
-It reduces cost, complexity, and time-to-market so businesses can be more competitive.
+Symbol is a feature-rich development platform that lets you build blockchain projects faster and more efficiently, so you can exceed expectations.
 
-Symbol offers flexible and secure customizable solutions through simple, yet powerful, built-in features.
-Developers are able to easily integrate blockchain technology into their projects or create powerful dApps using Symbol’s REST API.
-This allows creating a variety of architectural solutions with lightweight code efficiently, in any programming language.
-
-Symbol provides access to both public and private blockchains, individually or as a hybrid, providing options to best suit your company’s needs.
-
-**NOTE**: At the time of writing, |codename|'s **MAIN_NET** has not been released. The network specification, documentation, and tooling presented may suffer variations before the network launch. See here the complete roadmap for the public launch: [Symbol Launch Plan](https://blog.nem.io/nem-ecosystem-symbol-launch-plan/).
+Symbol uses familiar **APIs** to interface with your existing platform so the learning curve is lower and you can develop your way.
```

Comment:
Symbol uses familiar **APIs** so you can easily interface with your existing platform.